src/CMakeLists.txt

```diff
@@ -51,12 +51,13 @@ message( STATUS "Configuring Level 1 packages --" )
 add_dir_if_exists( ds++ )
 
 # Only test ds++ if CI_CLANG_TIDY is running
-if( NOT CI_CLANG_TIDY )
 
 # Level 2
 message(" ")
 message( STATUS "Configuring Level 2 packages --" )
 add_dir_if_exists( c4 )           # needs ds++
+
+if( NOT CI_CLANG_TIDY )
```

Comment:
Instruct the nightly clang-tidy regression that runs on ccscs1 to omit checking all directories listed after this line.